src/ui/pages/Settings/Settings.tsx

```diff
@@ -126,6 +129,63 @@ function SettingsMain() {
             },
             {
               key: 8,
+              pad: false,
+              style: { paddingInline: 0 },
+              component: (
+                <SurfaceItemAnchor
```

Comment:
This line should be shown only to non-premium wallets. However, it is not obvious how to check premium status in global menu related to all addresses.